Sources/TidepoolKit/TAPI.swift

```diff
@@ -766,8 +766,8 @@ public actor TAPI {
     /// - Parameters:
     ///   - start: the start date of the period that contains the log entries
     ///   - end: the end date of the period that contains the log entries
-    /// - Returns: A list of ``TDeviceLogsMetadata`` structures
-    public func listDeviceLogs(start: Date, end: Date) async throws -> [TDeviceLogsMetadata] {
+    /// - Returns: An optional list of ``TDeviceLogsMetadata`` structures. If the list is empty, `nil` is returned
```

Comment:
After some testing, it looks like the backend is returning "null", which could be considered valid json. 

Nit: I think if we are wanting to return something that means "empty list" from this method, it would be better to return an empty list. Otherwise callers need to wonder what the difference between [] and nil is, and handle both.